jobs/list.ts

```diff
@@ -36,4 +37,7 @@ export const allJobs: CSCronJob[] = [
     { cronTime: '00 00 10 01 * *', jobFunction: postStatisticsToSlack, name: 'postStatisticsToSlack' },
     // Every night, trigger actions for courses that ended yesterday
     { cronTime: '00 00 10 * * *', jobFunction: notificationsEndedYesterday, name: 'notificationsEndedYesterday' },
+    // Revalidate streaks are used to revalidate if streaks are still meeting their conditions and updates them accordingly
+    { cronTime: '00 00 02 * * 1', jobFunction: () => revalidateStreaks(StreakTimeFrame.WEEK), name: 'weeklyStreakRevalidation' },
+    { cronTime: '30 01 1 * *', jobFunction: () => revalidateStreaks(StreakTimeFrame.MONTH), name: 'monthlyStreakRevalidation' },
```

Comment:
same question here: every first day of a month at 1.30 a.m.? 😄

Reply:
Also correct! But as you mentioned this. I missed a 00 at the start. The configuration of cronTime works like this:

```
field          allowed values
-----          --------------
second         0-59
minute         0-59
hour           0-23
day of month   1-31
month          1-12 (or names, see below)
day of week    0-7 (0 or 7 is Sunday, or use names)
```